bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -2276,4 +2277,36 @@ private function addGlossaryNode(ArrayNodeDefinition $rootNode): void
                             ->useAttributeAsKey('name')
                             ->prototype('scalar');
     }
+    
+    /**
+     * @param ArrayNodeDefinition $rootNode
+     */
+    private function addNotificationsNode(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('notifications')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('check_new_notification')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->booleanNode('active')
```

Comment:
I'd prefer `canBeDisabled()` here, to have it consistent with other options.

Reply:
updated.